docs/sources/release-notes/release-notes-7-3-3.md

```diff
@@ -0,0 +1,20 @@
++++
+title = "Release notes for Grafana 7.3.3"
+[_build]
+list = false
++++
+
+<!-- Auto generated do not edit -->
+
+# Release notes for Grafana 7.3.3
+
+### Features / Enhancements
```

Comment:
I thought this was "Features and enhancements"... and "Bug Fixes" was "Bug fixes" upstream...

Reply:
yea, strange, must have copied from the wrong source